src/olympia/conf/dev/settings.py

```diff
@@ -261,7 +261,19 @@
         'redirect_url': 'https://amo.dev.mozaws.net/fxa-authenticate',
         'scope': 'profile',
     },
+    'development': {
```

Comment:
Names are hard but I would rather see something here like `local_development` or `local_dev` because I thought "development" meant the `addons-dev.allizom.org` server at first.